src/v/test_utils/runfiles.cc

```diff
@@ -22,7 +23,9 @@ get_runfile_path([[maybe_unused]] std::string_view path) {
         runfiles.reset(
           Runfiles::CreateForTest(BAZEL_CURRENT_REPOSITORY, &error));
     } else {
-        runfiles.reset(Runfiles::Create("", BAZEL_CURRENT_REPOSITORY, &error));
+        auto argv0 = std::filesystem::canonical("/proc/self/exe");
```

Comment:
Seems like this would be a good value to cache forever, like `static const argv0 = ...` ?

Reply:
good idea, done